my-app/src/components/Header/index.tsx

```diff
@@ -0,0 +1,303 @@
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import Menu from '@mui/material/Menu';
+import Container from '@mui/material/Container';
+import PersonIcon from '@mui/icons-material/Person';
+import Tooltip from '@mui/material/Tooltip';
+import MenuItem from '@mui/material/MenuItem';
+import AssessmentIcon from '@mui/icons-material/Assessment';
+import ToggleButton from '@mui/material/ToggleButton';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+import { Link } from 'react-router-dom';
+import { useAppSelector, RootState, useAppDispatch } from 'store/store';
+import { logout } from 'store/auth/auth-slice';
+import { useTranslation, Trans } from 'react-i18next';
+import useScrollTrigger from '@mui/material/useScrollTrigger';
+import theme from 'components/Theme';
+import PersonAddAlt1Icon from '@mui/icons-material/PersonAddAlt1';
+import LogoutIcon from '@mui/icons-material/Logout';
+
+const lngs = {
+  en: { nativeName: 'English' },
+  ru: { nativeName: 'Russia' },
+};
+
+function Header() {
+  const trigger = useScrollTrigger({
+    disableHysteresis: true,
+  });
+  const { t, i18n } = useTranslation();
+  const userId: string = useAppSelector((state: RootState) => state.rootReducer.authReducer?.userId);
+  const dispatch = useAppDispatch();
+  const [anchorElUser, setAnchorElUser] = React.useState<null | HTMLElement>(null);
+  const [alignment, setAlignment] = React.useState('web');
+
+  const handleChange = (event: React.MouseEvent<HTMLElement>, newAlignment: string) => {
+    setAlignment(newAlignment);
+  };
+  const handleOpenUserMenu = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorElUser(event.currentTarget);
+  };
+
+  const handleCloseUserMenu = () => {
+    setAnchorElUser(null);
+  };
+
+  return (
+    <AppBar
+      position="fixed"
+      sx={{
+        backgroundColor: trigger ? theme.palette.primary.light : 'primary',
+        boxShadow: trigger ? 'none' : 'default',
+      }}
+    >
+      <Container maxWidth="xl">
+        <Toolbar disableGutters>
+          <Link to="/">
+            <AssessmentIcon sx={{ display: 'flex', mr: 1, color: theme.palette.secondary.main }} />
+          </Link>
+          <Typography
+            variant="h6"
+            noWrap
+            component="a"
+            href="/"
+            sx={{
+              mr: 2,
+              display: { xs: 'none', md: 'flex' },
+              fontFamily: 'monospace',
+              fontWeight: 700,
+              letterSpacing: '.2rem',
+              color: 'inherit',
+              textDecoration: 'none',
```

Comment:
I prefer to write css in external file, not inside js file

Reply:
That is Material UI style, witch we used. I prefer it to. That's why I used preprocessor in board page. But my teammates do that according to recommendations of Material UI.

Reply:
typography component has classes property for such cases